modules/notes.js

```diff
@@ -75,8 +76,13 @@ const worker = async ({ channel, response_url, ts, text }) => {
   // filter out any other message that could be conversation including parent note
   const updates = thread.filter(({ text }) => {
     // get the first word of the block of text and check if it is an action
-    const { action } = text.match(/(?<action>[^/-]\S+)/).groups
-    return action.toLowerCase().match(/did|doing|issues|todo/)
+    const match = text.match(/(?<action>[^/-]\S+)/)
+
+    // TODO: extract info when hugo is used to add updates as match is null, temp fix
```

Comment:
_minor_ this reminds me of another error not handled:  if any of the comments in the thread is an empty string (no message), the command will error. It happened once with you, I guess I even asked you to delete the comment but I don't recall the details. It is similar to the hugo thing where the payload comes with an empty message

Reply:
there's also the error that is the link is not a parent message, slack can't process it (slack limitation). We could send a message to inform the user in this case too maybe

Reply:
righttt. we can do that in another PR where it can be about error handling (i.e. not parent message which you can tell bcuz the URL would have /thread or have a query param `thread_ts` if its a thread message + when message is null)